app/api_routes.js

```diff
@@ -886,5 +886,15 @@ routes.get('/api/*', (req, res) => {
     .status(404)
     .json({ status: 404, error: 'Not found. Did you mispell something?' })
 })
+routes.post('/api/*', (req, res) => {
+  res
+    .status(404)
+    .json({ status: 404, error: 'Not found. Did you mispell something?' })
+})
+routes.put('/api/*', (req, res) => {
+  res
+    .status(404)
+    .json({ status: 404, error: 'Not found. Did you mispell something?' })
+})
```

Comment:
We can cut down on repetition AND cover all HTTP verbs (including ones not handled here, like `DELETE` or `OPTIONS`) with [`app.all()` as the handler](https://expressjs.com/en/4x/api.html#app.all).